kura/org.eclipse.kura.log.filesystem.provider/src/main/java/org/eclipse/kura/log/filesystem/provider/KuraLogLineParser.java

```diff
@@ -106,13 +110,20 @@ private void parseKuraAuditLog() {
 
     private LogEntry generateLogEntry() {
         Map<String, Object> entryProperties = new HashMap<>();
-        entryProperties.put("_SOURCE_REALTIME_TIMESTAMP", instance.timestamp);
         entryProperties.put("_PID", instance.pid);
         entryProperties.put("MESSAGE", instance.message);
         entryProperties.put("PRIORITY", instance.priority);
         entryProperties.put("SYSLOG_IDENTIFIER", instance.syslogIdentifier);
         entryProperties.put("_TRANSPORT", instance.transport);
         entryProperties.put("STACKTRACE", instance.stacktrace);
+        
+        if (!DEFAULT_TIMESTAMP.equals(instance.timestamp)) {
+            try {
+                return new LogEntry(entryProperties, Long.parseLong(instance.timestamp.substring(0, 13)));
```

Comment:
Kura log timestamps are strings in form "2022-07-11T09:32:06,540". `Long.parseLong` will fail every time.

We could do something like that:
```
try {
    SimpleDateFormat dateFormat = new SimpleDateFormat("yyyy-MM-ddThh:mm:ss,SSS");
    Date parsedDate = dateFormat.parse(instance.timestamp);
    return new LogEntry(entryProperties, new Timestamp(parsedDate));
} catch ...
```